gm4_mob_curing/data/gm4_mob_curing/functions/cure/mooshroom.mcfunction

```diff
@@ -1,8 +1,9 @@
-#@s = cows with slowness needing conversion at @s
-#run from main
+# @s = cows with slowness needing conversion at @s
+# run from main
+
 summon mooshroom
 particle explosion ~ ~.5 ~ 1 .3 1 0 1
 playsound minecraft:entity.zombie_villager.converted neutral @a[distance=..8]
-execute as @e[type=item,limit=1,nbt={Item:{id:"minecraft:red_mushroom"}},distance=..3,sort=nearest] run function gm4_mob_conversion:consume_item
-execute as @e[type=item,limit=1,nbt={Item:{id:"minecraft:brown_mushroom"}},distance=..3,sort=nearest] run function gm4_mob_conversion:consume_item
+execute as @e[type=item,limit=1,nbt={Item:{id:"minecraft:red_mushroom"}},distance=..3,sort=nearest] run function gm4_mob_curing:cure/consume_item
+execute as @e[type=item,limit=1,nbt={Item:{id:"minecraft:brown_mushroom"}},distance=..3,sort=nearest] run function gm4_mob_curing:cure/consume_item
 data merge entity @s {DeathTime:19s,Health:0f,DeathLootTable:"minecraft:empty"}
```

Comment:
Are you sure this still works? I remember something about this being broken in 1.18 -- but I could be wrong.

_This comment also applies to the other curing flavors._

Reply:
This is regarding line 8, the data merge entity.   This still functions to kill, however no longer functions to ignore the death animation and cloud particle effect.  This will be changed to a off-screen teleport and kill @s